content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -195,6 +195,12 @@ authentication is needed.
                  gdrive_service_account_json_file_path path/to/file.json
    ```
 
+3. Share the Google Drive folders that you want to use with the service account.
+   Navigate to your Google Drive folder's sharing options and add the service
+   account as an editor:
```

Comment:
can it be a reader (depending on the use case?)

Reply:
Included this in the text. Kept it brief, but let me know if we want to elaborate on this more.